src/fiatconnect/FiatDetailsScreen.tsx

```diff
@@ -136,7 +141,10 @@ const FiatDetailsScreen = ({ route, navigation }: Props) => {
 
       const completeBody = {
         ...body,
-        ...implicitFields(fiatAccountSchema),
+        ...implicitParameters.reduce(
+          (prev, current) => ({ ...prev, [current.name]: current.value }),
```

Comment:
kinda nit here: i think just using a forEach() to build the object here is slightly more efficient and easier to read:

const implicitBody = {}
implicitParameters.forEach(param => {
   implicitBody[param.name] = param.value
}